src/components/CommentCard.jsx

```diff
@@ -1,14 +1,43 @@
-import React from 'react';
+import * as api from "../utils/api";
 import moment from 'moment';
+import React, { useState, useEffect,useContext } from "react";
+import {userContext} from "../utils/Context";
 
+function CommentCard({comment_id,author,body,votes,date}) {
+  const [deleteIcon,setDelete] = useState(null);
+  const [error,setError] = useState(null);
+  const [newBody,setBody] = useState(body);
+  const [issue, setIssue] = useState(false)
+  const {loggedInUser} = useContext(userContext)
+
+  const handleDelete = () => {
+    api.deleteCommentById(comment_id).then(()=>{ if(!issue){
+      setBody("Comment deleted")
+      setDelete(null)}}).catch((err) => {
+      setIssue(true)
+      setError("Hmmm what seems to be the officer problem ?")
+    });
+ 
+  };
+
+  useEffect(() => {
+    if (author === loggedInUser && !issue) {
+      setDelete(
+        <img  className="vte" onClick={handleDelete} src="https://img.icons8.com/ios-filled/50/ffffff/delete-sign--v1.png"/>
```

Comment:
images always should have an alt tag to explain their purpose, for screen readers. Something like alt="404 - page not found"